tests/helpers/app-fixture.js

```diff
@@ -0,0 +1,206 @@
+var fs = require('fs');
+var path = require('path');
+var stringUtil = require('ember-cli-string-utils');
+var execSync = require('child_process').execSync;
+var symlinkOrCopySync = require('symlink-or-copy').sync;
+
+var packageCache = require('./package-cache');
+
+var fixturify = require('fixturify');
+var quickTemp = require('quick-temp');
+var merge = require('ember-cli-lodash-subset').merge;
+
+function AppFixture(name, options) {
+  this.type = 'app';
+  this.name = name;
+  this.options = options || {
+    useGlobalPackages: true
+  };
+
+  this.fixture = {};
+  this._fixtureCache = {};
+  this.dirs = {};
+  quickTemp.makeOrRemake(this.dirs, 'self');
+
+  this.blueprintPath = path.resolve(__dirname, '../../blueprints/app');
+  this.blueprintOptionsShim = {
+    target: '',
+    entity: { name: name },
+    ui: {
+      writeLine: function() {}
+    },
+    project: {
+      isEmberCLIAddon: function() {
+        return false;
+      },
+      config: function() {
+        return {
+          usePodsByDefault: false
+        };
+      },
+      name: function() {
+        return name;
+      }
+    }
+  };
+
+  this.loadBlueprint();
+}
+
+AppFixture.prototype = {
+  _bowerInstall: function() {
```

Comment:
Is there a reason why these use `execSync`? I feel like you could create an API that this like this that should cut down the serial aspect of install deps for tests:

```
installDeps() {
  return Promise.all([this._bowerInstall(), this._npmInstall()]);
}
```